third_party/2and3/cryptography/hazmat/primitives/asymmetric/padding.pyi

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+from cryptography.hazmat.primitives.hashes import HashAlgorithm
+
+class AsymmetricPadding:
```

Comment:
```suggestion
class AsymmetricPadding(metaclass=ABCMeta):
```

Reply:
Is this correct, since there are no abstract methods? (there is only the "abstract" name property, but I believe abstract properties aren't supported)